src/router/index.tsx

```diff
@@ -23,12 +23,12 @@ export const AppRouter = () => {
 
   return (
     <Switch>
-      <Route exact path="/" component={AuctionsView} />
-      <Route exact path="/auctions" component={AuctionsView} />
-      <Route exact path="/auctions/:auctionId" component={AuctionView} />
-      <Route exact path="/auctions/fixed/:auctionId" component={FixedPriceAuctionView} />
+      <Route exact path="/" component={SalesView} />
+      <Route exact path="/sales" component={SalesView} />
+      <Route exact path="/sales/:saleId" component={SaleView} />
+      <Route exact path="/sales/fixed/:saleId" component={FixedPriceSaleView} />
```

Comment:
Create a sub-router `<Switch>` for sales. Determine the sale type, and then pass the route params to that view. There is no need to put the sale type in the URL. It will create scaling issues: with every new sale mechanism, the routing table has to be updated.